tests/reduce.spec.js

```diff
@@ -0,0 +1,30 @@
+import {reduce} from '../src';
+
+/* eslint-disable require-jsdoc */
+const testArray = [1, 2, 3, 4];
+
+describe('reduce', () => {
```

Comment:
I reckon it's worth including a test for executing `reduce` with an `initialValue` - see comment on `reduce` above